var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -6,50 +6,156 @@
 from spack import *
 
 
-class Hiop(CMakePackage):
+class Hiop(CMakePackage, CudaPackage):
     """HiOp is an optimization solver for solving certain mathematical
     optimization problems expressed as nonlinear programming problems.
     HiOp is a lightweight HPC solver that leverages application's existing
     data parallelism to parallelize the optimization iterations by using
     specialized linear algebra kernels."""
 
     homepage = "https://github.com/LLNL/hiop"
-    git      = "https://github.com/LLNL/hiop.git"
+    git = "https://github.com/LLNL/hiop.git"
 
+    # Most recent tagged snapshot is the preferred version when profiling.
+    version('0.3.99.3', tag='v0.3.99.3', preferred=True)
```

Comment:
```suggestion
    version('0.3.99.3', tag='v0.3.99.3')
```
latest version doesn't have to be marked preferred